### DISCUSSION TITLE
Why doesn’t the DOGE community host the DOGE blockchain on cryptoid.info?

### DISCUSSION THREAD

**Discussion by dogmcoin** [dogecoin/dogecoin#3641] [Q&A]

**Comment by eyemaginative** [comment]
For what purpose?  That site charges money to host block explorers and we already have block explorers.  Why pay somebody to host something when it isn't needed?

**Comment by dogmcoin** [comment]
To be honest, the current DOGE blockchain explorers are not very good.

**Comment by VishalRaut2106** [comment]
Great question! There are a few key reasons why the Dogecoin community likely prefers maintaining their own infrastructure rather than relying on cryptoid.info:

1. **Decentralization & Control**: One of the core principles of blockchain is decentralization. By hosting explorers through third-party services, the community would be dependent on external entities. Community-owned explorers ensure the Dogecoin network maintains independence and control over its infrastructure.

2. **Cost & Long-term Sustainability**: While some services offer free tiers, relying on paid hosting services can become expensive over time and leaves the community vulnerable to price changes or service discontinuation. Community-run infrastructure avoids these recurring costs and vendor lock-in.

3. **Privacy & Trust**: Third-party explorers can track user behavior and data. Community-run explorers provide better privacy for users and build trust within the ecosystem.

4. **Customization**: Community explorers can be customized specifically for Dogecoin's needs and values, rather than being generic solutions.

**Regarding Explorer Quality**: You make a fair point about current explorer quality. Rather than outsourcing to cryptoid.info, the community could focus on:
- Improving existing explorers through contributions and funding
- Creating new community-developed explorers
- Attracting developers to enhance exploration tools

If you're interested in this, consider contributing to existing Dogecoin explorer projects or proposing improvements on the community forum!
